src/settings.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "./settings.schema.json",
```

Comment:
this adds autocomplete for settings in editors that support it like vscode etc. for properties specified in the schema.